frontend/src/app/navbar/navbar.component.html

```diff
@@ -1,22 +1,10 @@
-<nav class="navbar navbar-light bg-light" style="margin-bottom: 50px;">
-  <a class="navbar-brand" href="#">
-    <img src="../../assets/logo.png" width="250" height="115" alt="Bulwark Logo" /></a>
-  <div class="mx-auto">
-    <div *ngIf="loaderService.isLoading | async" role="status">
-      <p-progressSpinner [style]="{width: '75px', height: '75px'}"></p-progressSpinner>
-    </div>
-  </div>
-  <div class="nav-item dropdown" *ngIf="loggedIn$ | async as loggedIn">
-    <a class="nav-link dropdown-toggle" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true"
-      aria-expanded="false">
-      Menu
+<div *ngIf="loaderService.isLoading | async" role="status">
+  <p-progressSpinner [style]="{width: '75px', height: '75px'}"></p-progressSpinner>
+</div>
+<p-menubar [model]="menuItems" *ngIf="loggedIn$ | async as loggedIn">
```

Comment:
<img width="739" alt="Screen Shot 2020-10-18 at 10 34 31 AM" src="https://user-images.githubusercontent.com/1582016/96370932-89523a00-112d-11eb-8ee1-f0eff3f7aece.png">
This conditional will completely hide the menu bar when the user logs out.  Instead, we need it so the menu items are hidden when the user is logged out.

<img width="1044" alt="Screen Shot 2020-10-18 at 10 24 36 AM" src="https://user-images.githubusercontent.com/1582016/96370539-2c09b900-112c-11eb-935f-8378a0ee4e8f.png">